src/tlo/methods/malaria.py

```diff
@@ -1334,6 +1491,39 @@ def apply(self, population):
                     data=prev,
                     description='Prevalence malaria cases')
 
+        # ------------------------------------ CO-INFECTION PREVALENCE ------------------------------------
+        if "Hiv" in self.sim.modules:
+            # number of people with both HIV and clinical/severe malaria
+            # output is malaria prevalence in HIV pop
+            coinfection_num = len(
+                df[df.is_alive & (df.ma_inf_type != 'none') & df.hv_inf]
+            )
+
+            # hiv population
+            hiv_infected = len(
+                df[df.is_alive & df.hv_inf]
+            )
+
+            # prevalence of malaria in HIV population
+            prev_malaria_in_hiv_population = coinfection_num / hiv_infected
```

Comment:
Careful about division of values than could be 0. Use safety around this: eg

X if y > 0 else 0

Reply:
changed